docs/api/app.md

```diff
@@ -349,6 +349,10 @@ Returns:
 * `authenticationResponseDetails` Object
   * `url` URL
   * `pid` number
+  * `isMainFrame` boolean - Indicates whether the request is for the main frame.
```

Comment:
IMO we should expose the frame, not just isMainFrame

Reply:
This is not my idea. It is only docs about: https://github.com/electron/electron/pull/46630/files#diff-84fba58576f3f13f6f3e6269cf65da405fd7b548c2a077f17114f68103ed9572R69

Maybe can change to `isRequestForMainFrame`, but this issue is not about it.

Reply:
Unfortunately this issue is about it, the parameter before was undocumented and not part of formal API. This PR makes it part of formal API and therefore we must determine if that API is appropriate or not. This comment is blocking unless @electron/wg-api discusses and concludes differently

Reply:
This property is also a lie. `isMainFrame` maps the `is_request_for_primary_main_frame_navigation` in chromium code. Which have subtle yet significant differences in meaning, the request being from the main frame vs being caused by a navigation in the main frame implies that you can make a request in the main frame **not** for a navigation and trigger this event with `false`

Reply:
Ok. And what now? This issue was only about documentation. If you (or other else) want change more source code, that must do someone with more skills about Electron/Chromium, not me.